dune_api_scripts/queries.py

```diff
@@ -130,25 +130,25 @@ def build_query_for_affiliate_data(startDate, endDate):
         day,
         referrer,
         sum(cowswap_usd_volume) as "total_referred_volume",
-        sum(number_of_trades) as "nr_of_referrals" 
+        ARRAY_AGG(DISTINCT Replace(user_stats_of_gp.owner, '\\x', '0x')) as "referrals"
```

Comment:
Today I learned: 

[ARRAY_AGG](https://www.postgresqltutorial.com/postgresql-aggregate-functions/postgresql-array_agg-function/)

How did you stumble upon this?